src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+for (const li of tree.querySelectorAll('li')) {
+  const span = document.createElement('span');
+
+  span.textContent = li.firstChild.textContent;
+  li.firstChild.textContent = '';
+  li.prepend(span);
+}
+
+tree.onclick = function (e) {
```

Comment:
better to add the `addEventListener` instead of the `onclick`(we usually use it in html)